pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -1,479 +1,497 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-	<!-- 
-		Microsoft ResX Schema
+  <!-- 

```

Comment:
Please make this change less extensive.

Reply:
Visual Studio's treatment of resx and designer-managed files is infuriating. They just get rearranged for no damn (discernable) reason. You practically have to text-edit the things for simple changes, or give up on being able to easily track the real changes (rather than reorderings).

Reply:
Kinda makes you wonder about their interior change control processes, doesn't it?

Reply:
It does indeed. Maybe they use some special sort-before-diffing engine for resx and designer files, like this guy did back in 2009: https://www.codeproject.com/Articles/37022/Solving-the-resx-Merge-Problem - but Googling reveals surprisingly little about dealing with the problem.